cluster/gce/config-default.sh

```diff
@@ -55,6 +58,11 @@ if [[ "${NODE_OS_DISTRIBUTION}" == "cos" ]]; then
     NODE_OS_DISTRIBUTION="gci"
 fi
 
+# GPUs supported in GCE do not have compatible drivers in Debian 7.
+if [[ "${NODE_OS_DISTRIBUTION}" == "debian" ]]; then
```

Comment:
We are blacklisting debian for GPU support.  Should we instead whitelist gci?  May be better if we want to add more OS distributions (ubuntu?) in the future.